### ISSUE TITLE
Styles not showing up in description page for high contrast theme

### ISSUE THREAD

**Issue by Tyriar** [microsoft/vscode-pull-request-github#897]
Microsoft/vscode#67509

The first time I viewed it, it was like this:

![image](https://user-images.githubusercontent.com/2193314/51999112-e811e480-246e-11e9-9033-e6d3158fc099.png)

The next:

![image](https://user-images.githubusercontent.com/2193314/51999129-f102b600-246e-11e9-84e5-96cdbcd2dec1.png)

I am seeing this in the console so it could be related:

```
ERR Cannot read property 'prNumber' of undefined: TypeError: Cannot read property 'prNumber' of undefined
    at c.<anonymous> (/home/daniel/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-1.30.0/media/extension.js:1:258479)
    at Generator.next (<anonymous>)
    at n (/home/daniel/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-1.30.0/media/extension.js:1:256528)
    at new Promise (<anonymous>)
    at n (/home/daniel/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-1.30.0/media/extension.js:1:256305)
    at c.update (/home/daniel/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-1.30.0/media/extension.js:1:258294)
    at constructor._panel.onDidChangeViewState.e (/home/daniel/.vscode-insiders/extensions/github.vscode-pull-request-github-insiders-1.30.0/media/extension.js:1:257083)
    at e.fire (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:44:133)
    at e.$onDidChangeWebviewPanelViewState (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:628:204)
    at t._doInvokeHandler (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:644:118)
    at t._invokeHandler (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:643:744)
    at t._receiveRequest (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:642:322)
    at t._receiveOneMessage (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:641:155)
    at /usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:638:960
    at /usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:38:994
    at e.fire (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:44:113)
    at a (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:213)
    at Socket.f._socketDataListener (/usr/share/code-insiders/resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:170:434)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:279:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at Pipe.onread (net.js:636:20)
```

Output:

```
[Info] /home/daniel/.ssh/config: ENOENT: no such file or directory, open '/home/daniel/.ssh/config'
[Info] Looking for git repository
[Info] Git repository found, initializing review manager and pr tree view.
[Info] No GitHub remotes found
[Info] Review> no matching pull request metadata found for current branch master
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch master
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch master
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch master
[Info] GitHubRepository> The remote branch for this PR was already deleted.
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch i_was_here
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch i_was_here
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch i_was_here
[Info] GitHubRepository> The remote branch for this PR was already deleted.
[Info] PullRequestGitHelper> associate i_was_here with Pull Request #50
[Info] Found GitHub remote
[Info] Review> current branch i_was_here is associated with pull request #50
[Info] Review> {"message":"Not Found","documentation_url":"https://developer.github.com/v3/repos/commits/#compare-two-commits"}
[Info] Review> register comments provider
[Info] Review> display pull request status bar indicator and refresh pull request tree view.
[Info] GitHubRepository> The remote branch for this PR was already deleted.
[Info] Found GitHub remote
[Info] Review> no matching pull request metadata found for current branch master
[Info] GitHubRepository> The remote branch for this PR was already deleted.
```

**Comment by Tyriar** [comment]
Also syntax coloring and monospace fonts aren't working in the diff above
